### PR TITLE
chore: make layout design previews show RecyclerView list-items

### LINKED ISSUE #1039: Add layout previews for listitems of RecyclerViews
**Describe the chore**
Currently, the layout previews don't show the list-items of `RecyclerViews`. Their layouts have to be opened separately. This increases the time one needs to spend to view the entire layout of the screen. An example is the `fragment_search_results.xml` file. Fixing this will allow people to view the entire screen's layout by simply opening a single layout file.

**Would you like to work on the issue?**
- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by AnEnigmaticBug**
Fixes #1039 

Changes: Earlier, layout previews of layouts with RecyclerViews didn't display the list-items. This PR adds tools:listItem and tools:itemCount attributes which fix this. Now, viewing the preview of a layout shows the list-items too. This makes it more convenient to view layout previews.

### CODE REVIEW

File: app/src/main/res/layout/fragment_attendee.xml [hunk 1]
```diff
@@ -197,8 +197,8 @@
                 <com.google.android.material.textfield.TextInputLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_marginBottom="@dimen/padding_medium"
-                    android:layout_marginTop="@dimen/padding_medium">
+                    android:layout_marginTop="@dimen/padding_medium"
+                    android:layout_marginBottom="@dimen/padding_medium">
```
Comment (addiegupta) [inline app/src/main/res/layout/fragment_attendee.xml]: I dont think this change is necessary

File: app/src/main/res/layout/fragment_attendee.xml [hunk 2]
```diff
@@ -211,8 +211,8 @@
...
```
Comment (addiegupta) [inline app/src/main/res/layout/fragment_attendee.xml]: I dont think this change is necessary

File: app/src/main/res/layout/fragment_tickets.xml [hunk 3]
```diff
@@ -113,11 +113,14 @@
                     android:textSize="@dimen/text_size_large"
                     tools:text="Qty" />
             </LinearLayout>
+
             <androidx.recyclerview.widget.RecyclerView
```
Comment (addiegupta) [inline app/src/main/res/layout/fragment_tickets.xml]: Unnecessary change

File: app/src/main/res/layout/fragment_attendee.xml (hunk 2 above)
Comment (AnEnigmaticBug) [reply app/src/main/res/layout/fragment_attendee.xml]: I guess I accidentally ran the code-formatted in Android Studio. I'll remove these changes right away.

File: app/src/main/res/layout/fragment_tickets.xml (hunk 3 above)
Comment (AnEnigmaticBug) [reply app/src/main/res/layout/fragment_tickets.xml]: I'll remove it

File: app/src/main/res/layout/fragment_attendee.xml (hunk 1 above)
Comment (AnEnigmaticBug) [reply app/src/main/res/layout/fragment_attendee.xml]: I'll remove this.
